packages/crypto/src/crypto.ts

```diff
@@ -0,0 +1,405 @@
+import { p256 } from "@noble/curves/p256";
+import * as hkdf from "@noble/hashes/hkdf";
+import { sha256 } from "@noble/hashes/sha256";
+import { gcm } from "@noble/ciphers/aes";
+import { uint8ArrayToHexString } from "@turnkey/encoding";
+import * as bs58check from "bs58check";
+
+import {
+  AES_KEY_INFO,
+  HPKE_VERSION,
+  IV_INFO,
+  LABEL_EAE_PRK,
+  LABEL_SECRET,
+  LABEL_SHARED_SECRET,
+  SUITE_ID_1,
+  SUITE_ID_2,
+} from "./constants";
+
+interface HPKEDecyptParams {
+  ciphertextBuf: Uint8Array;
+  encappedKeyBuf: Uint8Array;
+  receiverPriv: string;
+}
+
+interface KeyPair {
+  privateKey: string;
+  publicKey: Uint8Array | string;
+  publicKeyUncompressed?: string;
+}
+
+/**
+ * Get PublicKey function
+ * Derives public key from Uint8Array or hexstring private key
+ *
+ * @param {Uint8Array | string} privateKey - The Uint8Array or hexstring representation of a compressed private key
+ * @param {boolean} isCompressed - true by default, specifies whether to return a compressed or uncompressed public key
+ * @returns {<Uint8Array>} - The public key in Uin8Array representation.
+ */
+
```

Comment:
let's remove this empty line for consistency

Reply:
We really need a whitespace linter 😅